plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -2088,17 +1994,6 @@ static int load_fleet_config(struct flb_in_calyptia_fleet_config *ctx)
         else if (exists_new_fleet_config(ctx) == FLB_TRUE) {
             return execute_reload(ctx, new_fleet_config_filename(ctx));
         }
-        else {
```

Comment:
The branches above will load either current or new config. No need to pick otherwise random config and a subsequent PR will refactor this function a bit anyway.